docs/advanced_usage/9_parallelism.md

```diff
@@ -1,41 +1,76 @@
 # Parallelism
 
-SMAC supports multiple workers natively via Dask. Just specify ``n_workers`` in the scenario and you are ready to go. 
-
+To facilitate fast execution, SMAC supports executing multiple workers simultaneously via [Dask](https://www.dask.org/). Using this functionality, splits SMAC into a main, and executor jobs.
+The main job handles the optimization process, and coordinates the executor jobs. The executors are queried with the target function and hyperparameter configurations, execute them, and return their result. 
 
 !!! note
     
     Please keep in mind that additional workers are only used to evaluate trials. The main thread still orchestrates the
     optimization process, including training the surrogate model.
 
-
 !!! warning
 
-    Using high number of workers when the target function evaluation is fast might be counterproductive due to the 
-    overhead of communcation. Consider using only one worker in this case.
-
+    When using multiple workers, SMAC is not reproducible anymore.
 
-!!! warning
 
-    When using multiple workers, SMAC is not reproducible anymore.
+## Parallelizing Locally
 
+To utilize parallelism locally, that means running workers on the same machine as the main jobs, specify the ``n_workers`` keyword when creating the scenario.
+```python
+Scenario(model.configspace, n_workers=5)
+``` 
 
-## Running on a Cluster
 
-You can also pass a custom dask client, e.g. to run on a slurm cluster.
-See our [parallelism example](../examples/1%20Basics/7_parallelization_cluster.md).
+## Parallelizing on SLURM
 
-!!! warning
+To utilize this split of main and execution jobs on a [SLURM cluster](https://slurm.schedmd.com/), SMAC supports manually specifying a [Dask](https://www.dask.org/) client.
+This allows executing the target function on dedicated SLURM jobs that are necessarily configured with the same hardware requirements,.
 
-    On some clusters you cannot spawn new jobs when running a SLURMCluster inside a
-    job instead of on the login node. No obvious errors might be raised but it can hang silently.
+!!! note
 
-!!! warning
+    While most SLURM clusters behave similarly, the example DASK client, might not work for every cluster. For example, some clusters only allow spawning new jobs 
+    from the login node.
 
-    Sometimes you need to modify your launch command which can be done with
-    `SLURMCluster.job_class.submit_command`.    
+To configure SMAC properly for each cluster, you need to know the ports which allow communication between main and worker jobs. The dask client is then created as follows:
```

Comment:
how can we find out that? or is this something the user sets?

Reply:
Its something the user sets afaik. There are default ports it queries, but for kisski they do not work